src/geometry.jl

```diff
@@ -94,23 +104,25 @@ function create_geometry(mechanism; show_inertias::Bool=false, randomize_colors:
             color = RGBA{Float64}(1, 0, 0, 0.5)
         end
         body = vertex_data(vertex)
+        frame = default_frame(mechanism, body)
         geometries = Vector{GeometryData}()
-        if show_inertias && !isnull(body.inertia) && get(body.inertia).mass >= 1e-3
+        if show_inertias && has_defined_inertia(body) && spatial_inertia(body).mass >= 1e-3
         # if show_inertias && !isroot(mechanism, body) && body.inertia.mass >= 1e-3
-            ellipsoid, tform = inertial_ellipsoid(body)
+            ellipsoid, ellipsoid_frame = inertial_ellipsoid(mechanism, body)
+            tform = AffineMap(find_fixed_transform(mechanism, ellipsoid_frame, frame))
```

Comment:
We actually don't need to do this anymore. I've changed the internals of DrakeVisualizer to operate on arbitrary frames instead of body names, so we no longer need to add a tform to each GeometryData. I'll make that change.